src/prelude/internals.mo

```diff
@@ -519,15 +519,8 @@ func @timer_helper() : async () {
   func Array_init<T>(len : Nat,  x : T) : [var T] {
     (prim "Array.init" : <T>(Nat, T) -> [var T])<T>(len, x)
   };
-  let now = (prim "time" : () -> Nat64)();
```

Comment:
This is weird because this file has no difference to master and to the base branch `luc/forwaring_pointers`.
Maybe a GitHub compare issue?